otcextensions/osclient/css/v1/snapshot.py

```diff
@@ -269,17 +240,19 @@ def get_parser(self, prog_name):
         parser.add_argument(
             '--period',
             metavar='<period>',
-            required=True,
-            help=('Time when a snapshot is created every day.\n'
-                  'Time format is followed by the time zone HH:mm z.\n'
-                  'for example, 00:00 GMT+08:00 and 01:00 GMT+08:00.')
+            default='01:00 GMT+01:00',
+            help=('Time when a snapshot is created every day. '
+                  'Time format is followed by the time zone HH:mm z. '
+                  'For example, 00:00 GMT+08:00 and 01:00 GMT+08:00. '
+                  '(Default Value: 01:00 GMT+01:00)')
         )
         parser.add_argument(
             '--keep-days',
             metavar='<keep_days>',
             type=int,
-            required=True,
-            help=('Number of days that a snapshot can be retained.')
+            default=15,
+            help=('Number of days that a snapshot can be retained. '
+                  '(Default Value: 15).')
```

Comment:
why you defaulted this parameters? 15 for keep_days it is a big period.

Reply:
Removed default value for --keep-days